### PR TITLE
Update Old Pages/Printers.md

### LINKED ISSUE #20: Printer & Carbon Processor Pages
(should merge carbon processors into printers)

### PR DESCRIPTION
**Pull request by Shady9879**
Added pictures and text formatting. Merged the carbon processor to this page. Closes #20.

### CODE REVIEW

**Review by hafarooki — CHANGES_REQUESTED** [review]
Looks like you forgot to actually delete the processors page

File: Old Pages/Printers.md [hunk 1]
```diff
@@ -1,64 +1,130 @@
...
-1 Printers
-1.1 Printer Multiblock:
-1.1.1 Carbon Printer
-1.1.2 Technical Printer
-1.1.3 Armor Printer
-1.1.4 Glass Printer
-1.1.5 Universal Printer Materials List
-Printers
-Printers do what alchemists couldn't - convert cobblestone into other blocks! You can use printers to generate carbyne (concrete/ship hull blocks), technical blocks (sponges), armor (terracotta/stained clay), and glass.
+# Printers and Processors
```
Comment (hafarooki) [inline Old Pages/Printers.md]:
```suggestion
# Printers
```

File: Old Pages/Printers.md [hunk 2]
```diff
@@ -1,64 +1,130 @@
...
+*Printers* are machines that turn cobblestone and energy into various materials.
+- Concrete powder, glass, sponge, and terracotta.
```
Comment (hafarooki) [inline Old Pages/Printers.md]:
```suggestion
```

File: Old Pages/Printers.md [hunk 3]
```diff
@@ -1,64 +1,130 @@
...
 
-To use the printer, ensure the sign has power. Then, put a focusing lens (prismarine crystals) in the top slot. Then, put cobblestone in the fuel slot. The printer should now begin converting.
+*Carbon processors* simply process raw concrete powder into concrete.
```
Comment (hafarooki) [inline Old Pages/Printers.md]: move this to the carbon printer section

File: Old Pages/Printers.md [hunk 4]
```diff
@@ -1,64 +1,130 @@
...
 
-Printer Multiblock:
-877cd4a34fb7981c6b76d506a82ad622.png b885e23103a52140c40d4ef9bd8eb9cb.png 646b834a73eefd71bb58c27b25dfad17.png f9157506ea35b3098b7aea138a41607c.png bb7332840871867bdb8ba3d3f579673e.png
+## Printers
```
Comment (hafarooki) [inline Old Pages/Printers.md]:
```suggestion
```

File: Old Pages/Printers.md [hunk 5]
```diff
@@ -1,64 +1,130 @@
...
 
-You replace the red blocks with the "machinery blocks", the blue block with "core block", and the white block with the "product block".
+### Using a printer:
```
Comment (hafarooki) [inline Old Pages/Printers.md]:
```suggestion
## Using a printer:
```

File: Old Pages/Printers.md [hunk 6]
```diff
@@ -1,64 +1,130 @@
...
+-  Ensure the sign has power (accepts power through noteblock).
+- Then, put a focusing lens (prismarine crystals) in the top slot.
+- Next, put cobblestone in the fuel slot.
+- The printer should now begin converting.
 
-NOTE: Copper blocks are only on ONE side which is ONLY on the LEFT of the sign with [Printer] on it.
+### Printer Multiblock:
```
Comment (hafarooki) [inline Old Pages/Printers.md]:
```suggestion
## Printer Multiblock
```

File: Old Pages/Printers.md [hunk 7]
```diff
@@ -1,64 +1,130 @@
...
 
-Generates: 2x Graxene Powder (Concrete Powder)
+![printerfront]
+![printerback]
+![printerleft]
+![printertop]
+![printerright]
 
-Machinery Blocks: Sponge
+- NOTE: Copper blocks are only on the LEFT of the sign with [Printer] on it.
```
Comment (hafarooki) [inline Old Pages/Printers.md]:
```suggestion
> Copper blocks are only on the LEFT of the sign with [Printer] on it.
```

File: Old Pages/Printers.md [hunk 8]
```diff
@@ -1,64 +1,130 @@
...
 
-Core Block: Magma
+*Universal Printer Materials List:*
```
Comment (hafarooki) [inline Old Pages/Printers.md]:
```suggestion
### Shared required materials
```

File: Old Pages/Printers.md [hunk 9]
```diff
@@ -1,64 +1,130 @@
...
+- 1x Note Block,
+- 2x Wires,
+- 3x Copper Blocks,
+- 4x Stained Glass Panes,
+- 6x Stained Glass,
+- 6x Iron Blocks.
+- The rest of the mats involved are the blocks that cause specific results.
 
-Product: Graxene Powder (Concrete Powder)
+### **Carbon Printer**
```
Comment (hafarooki) [inline Old Pages/Printers.md]:
```suggestion
### Carbon Printer
```

**Review by hafarooki — CHANGES_REQUESTED** [review]
Formatting on this one isn't very good, try to make it a bit more consistent and use markdown features properly

**Comment by Shady9879** [comment]
Made changes, for whatever reason I don't see the carbon processor page in my file repository?

**Comment by hafarooki** [comment]
IDK :P

File: Old Pages/Printers.md [hunk 10]
```diff
@@ -1,64 +1,127 @@
...
-To use the printer, ensure the sign has power. Then, put a focusing lens (prismarine crystals) in the top slot. Then, put cobblestone in the fuel slot. The printer should now begin converting.
+## Using a printer:
+-  Ensure the sign has power (accepts power through noteblock).
+- Then, put a focusing lens (prismarine crystals) in the top slot.
+- Next, put cobblestone in the fuel slot.
+- The printer should now begin converting.
 
-Printer Multiblock:
-877cd4a34fb7981c6b76d506a82ad622.png b885e23103a52140c40d4ef9bd8eb9cb.png 646b834a73eefd71bb58c27b25dfad17.png f9157506ea35b3098b7aea138a41607c.png bb7332840871867bdb8ba3d3f579673e.png
+## Printer Multiblock:
```
Comment (hafarooki) [inline Old Pages/Printers.md:10]: Remove colon

File: Old Pages/Printers.md [hunk 11]
```diff
@@ -1,64 +1,127 @@
...
+- Then, put a focusing lens (prismarine crystals) in the top slot.
+- Next, put cobblestone in the fuel slot.
+- The printer should now begin converting.
 
-Printer Multiblock:
-877cd4a34fb7981c6b76d506a82ad622.png b885e23103a52140c40d4ef9bd8eb9cb.png 646b834a73eefd71bb58c27b25dfad17.png f9157506ea35b3098b7aea138a41607c.png bb7332840871867bdb8ba3d3f579673e.png
+## Printer Multiblock: 
 
-You replace the red blocks with the "machinery blocks", the blue block with "core block", and the white block with the "product block".
+- Red blocks are replaced with the "machinery blocks".
```
Comment (hafarooki) [inline Old Pages/Printers.md:12]: Maybe use bold rather than quote

**Review by hafarooki — CHANGES_REQUESTED** [review]

**Comment by Shady9879** [comment]
Deleted and re downloaded and still don't see the carbon processor page.. If you still see it maybe you can delete that one please.

**Comment by hafarooki** [comment]
Did you close it without merging?

**Pull request closed without merging**